src/wwwroot/app/directives/thesaurus/thesaurus.directive.html

```diff
@@ -0,0 +1,46 @@
+<div class="errorMessage" ng-bind="message" ng-disabled="!message"></div>
+<table>
+   <caption ng-bind="::structure.thesaurusName | translate"></caption>
+   <tr><th ng-repeat="field in ::structure.fields | filter : isShowField" ng-bind="::field.label"></th></tr>
+   <tr ng-repeat = "topic in topics">
+      <td ng-repeat = "field in ::structure.fields | filter : isShowField" >
+         <span ng-if="!isEditTopic(topic)&&field.type==='text'" ng-bind="::topic[field.name]"></span>
```

Comment:
maybe will be a good idea to use `ng-disabled` instead of render two different element?
and styling disabled element as simple text?